docs/api-docs/store-management/shipping/use-real-time-carrier.md

```diff
@@ -0,0 +1,375 @@
+
+## Prerequisites
+- For shipping provider apps only: Merchant has installed the shipping provider app. 
+- You have obtained the required [OAuth Scopes](/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes): `Information & Settings`.
+- You or the merchant have created shipping zones. You can create zones with the [Create a shipping zone](/api-reference/store-management/shipping-api/shipping-zones/createashippingzones) endpoint.
+
+<!-- theme:info -->
+> #### Note
+> USPS, FedEx, and UPS carriers are native-integrations. For a full list of native integrations, see the [Real-Time Shipping Providers](https://support.bigcommerce.com/s/article/Setting-Up-a-Real-Time-Shipping-Quote-Shipping-Method#providers) article.
+
+## Manage connections to shipping carriers 
+
+When a merchant installs a shipping provider app, the carrier appears in the merchant's control panel under real-time shipping methods, but is not connected to the store by default. You must first connect the carrier to the store.
+
+![Connected vs Installed Carrier](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Installed%20vs%20Connected%20Carrier.png 'Connected versus Installed Carrier. USPS is connected, while FedEx is only installed.')  
+
+### Create a connection
+
+To set up a carrier using the API, connect it using the [Create a Carrier Connection](/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint. Make a request containing the ID of the carrier and any configuration fields that the carrier requires. All configuration fields are unique to each carrier. 
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{	
+  "carrier_id" : "endicia",
+  "connection": {
+      "account_id" : "example_id",
+      "pass_phrase" : "example_passphrase"
+  }
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example POST response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+If a carrier doesn’t require any connection settings, send an empty object for the `connection` property.
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "carrier_id": "example_carrier",
+  "connection": {}
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example POST response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+### Update a connection
+
+When you update a connection, use the same fields for a carrier as [Create a connection](#create-a-connection). 
```

Comment:
```suggestion
When you update a connection, use the same configuration fields as [Create a connection](#create-a-connection). 
```
make a choice about calling them `connection` fields vs. `configuration` fields, and stick with it.